src/components/editor-page/splitter/split-divider/split-divider.tsx

```diff
@@ -7,24 +7,61 @@
 import React from 'react'
 import styles from './split-divider.module.scss'
 import { testId } from '../../../../utils/test-id'
+import { Button } from 'react-bootstrap'
+import { ForkAwesomeIcon } from '../../../common/fork-awesome/fork-awesome-icon'
+
+export enum Corners {
```

Comment:
Corners is a bad term for this imho. I'd suggest using `SpliiterLocation`, `SpliterState` or something similiar…